feature-assets/src/main/java/io/novafoundation/nova/feature_assets/presentation/balance/search/AssetSearchFragment.kt

```diff
@@ -81,15 +89,17 @@ class AssetSearchFragment :
     override fun subscribe(viewModel: AssetSearchViewModel) {
         searchAssetSearch.searchInput.content.bindTo(viewModel.query, lifecycleScope)
 
+        viewModel.query.observe {
+            searchAssetList.post {
+                searchAssetList.layoutManager!!.scrollToPosition(0)
+            }
+        }
+
         viewModel.searchResults.observe { data ->
             searchAssetsPlaceholder.setVisible(data.isEmpty())
             searchAssetList.setVisible(data.isNotEmpty())
 
-            assetsAdapter.submitListPreservingViewPoint(
-                data = data,
-                into = searchAssetList,
-                extraDiffCompletedCallback = { searchAssetList.invalidateItemDecorations() }
-            )
+            assetsAdapter.submitList(data) { searchAssetList.invalidateItemDecorations() }
```

Comment:
I don't use `submitListPreservingViewPoint` since it affects items animation.
Instead of that I reset recyclerview scroll position each time search query is updated